docs/sql-ref-syntax-qry-select.md

```diff
@@ -151,6 +151,18 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { named_expression [ , ... ] }
 
      Specifies aliases for one or more source window specifications. The source window specifications can
      be referenced in the widow definitions in the query.
+     
+* **regex_column_names**
+
+     When `spark.sql.parser.quotedRegexColumnNames` is true, quoted identifiers (using backticks) in `SELECT`
+     statement are interpreted as regular expressions and `SELECT` statement can take regex-based column specification.
+     For example, below SQL will only take column `c`:
+
+     ```sql
+     SELECT `(a|b)?+.+` FROM (
+       SELECT 1 as a, 2 as b, 3 as c
+     ) src
```

Comment:
How about moving this example into the bottom of this page just like the other SQL references?

Reply:
> How about moving this example into the bottom of this page just like the other SQL references?

Here we just show a  example usage and look good. Only move this one SQL to bottom is strange and reader may ignore it(For example, me...).